### PR TITLE
introduce WebVR v1.0 API support

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: WebGLTemplates/WebVR/js/UnityWebVRManager.js [hunk 1]
```diff
@@ -1,93 +1,307 @@
...
+  function resizeCanvas () {
+    if (isPresenting()) {
+      if (isDeprecatedAPI) {
+        canvas.width = Math.max(eyeParamsL.renderRect.width * 2, eyeParamsL.renderRect.height);
+        canvas.height = Math.max(eyeParamsL.renderRect.height, eyeParamsL.renderRect.height);
+      } else {
+        // canvas.width = Math.max(eyeParamsL.renderWidth * 2, eyeParamsL.renderHeight);
+        // canvas.height = Math.max(eyeParamsL.renderHeight, eyeParamsL.renderHeight);
+        // TODO: Figure out how to properly mirror the canvas stereoscopically with the v1.0 API in Chromium.
+        // See https://github.com/toji/webvr-samples/blob/633a43e/04-simple-mirroring.html#L227-L231
```
Comment (cvan) [inline WebGLTemplates/WebVR/js/UnityWebVRManager.js:306]: /cc @caseyyee @toji

File: README.md [hunk 2]
```diff
@@ -38,6 +38,26 @@ Once your project finishes building, open the generated `index.html` in a [WebVR
 * [Instructions to set up Firefox Nightly with WebVR](http://mozvr.com/#start)
 
 
+## Local Development Notes
```
Comment (cvan) [inline README.md:41]: `TODO`: cleanup

File: WebGLTemplates/WebVR/js/UnityWebVRManager.js [hunk 3]
```diff
@@ -289,7 +289,7 @@
       initVrEventListeners();
       getEyeParameters();
       resizeCanvas();
-      update();
+      window.requestAnimationFrame(update);
```
Comment (caseyyee) [inline WebGLTemplates/WebVR/js/UnityWebVRManager.js]: This might cause issues?  We call it in raf method.

**Comment by gtk2k** [comment]
Thanks for PR !

**Pull request merged**